deps/graph-parser/src/logseq/graph_parser/block.cljs

```diff
@@ -142,34 +142,30 @@
 (defn- get-page-ref-names-from-properties
   [format properties]
   (let [page-refs (->>
-                     properties
-                     (remove (fn [[k _]]
-                               (contains? #{:background-color :background_color} (keyword k))))
-                     (map last)
-                     (map (fn [v]
-                            (cond
-                              (and (string? v)
-                                   (not (gp-mldoc/link? format v)))
-                              (let [v (string/trim v)
-                                    result (text/split-page-refs-without-brackets v {:un-brackets? false})]
-                                (if (coll? result)
-                                  (map text/page-ref-un-brackets! result)
-                                  []))
-
-                              (coll? v)
-                              (map (fn [s]
-                                     (when-not (and (string? v)
-                                                    (gp-mldoc/link? format v))
-                                       (text/page-ref-un-brackets! s))) v)
-
-                              :else
-                              nil)))
-                     (apply concat))
-        property-keys-page-refs (some->> properties
```

Comment:
I thought we were going with the blacklist/disallowed approach for built-in properties. To do that, I think we only need to modify this block e.g.:

```clojure
  property-keys-page-refs (some->> properties
                                         (map (comp name first))
                                         (remove string/blank?)
                                         (remove #{"id" "collapsed" "filters" "title" ...})
                                         (distinct))]
```

Reply:
Can you guys make it three lists: `(built-in-properties)`, `flashcards properties` and a custom list that can be specified by users in `config.edn`?

Its not always desirable to turn all properties values to pages. For example, I use a `difficulty` properties with values like `1, 2, ...`. I fell it would be odd to have a page with name `1`. 🐙

Reply:
Sounds like some basic property schema support